ci/common/build.sh

```diff
@@ -84,12 +84,11 @@ build_nvim() {
   fi
 
   # Invoke nvim to trigger *San early.
-  if ! (bin/nvim --version && bin/nvim -u NONE -e -c ':qall') ; then
-    asan_check "${LOG_DIR}"
+  if ! (bin/nvim --version && bin/nvim -u NONE -e -cq | cat -A) ; then
```

Comment:
`cat -A` does not work on osx.. fix in https://github.com/neovim/neovim/pull/10779